app/views/landing-pages/parents/PageParents.vue

```diff
@@ -1179,6 +1178,16 @@ export default {
   margin-bottom: 20px;
 }
 
+.concept-covered-tab.beginner img{
+  max-width: 92px !important;
```

Comment:
The selector seems specific enough that `!important` shouldn't be required?

Reply:
Removed `!important` from here.